### DISCUSSION TITLE
Asymmetric impact

### DISCUSSION THREAD

**Discussion by Tomoya-Iizawa** [scikit-hep/cabinetry#464] [Q&A] [answered]
Hi,
I am seeing the asymmetric impacts in the ranking plot as attached figure.
![asymm](https://github.com/scikit-hep/cabinetry/assets/61150311/4505168a-91d1-46ec-9b97-e2768b89f661)
This is from statistical error, but can be seen for systematic sources as well.
Do you have some idea why this happens? General idea is also welcome.

Best,
Tomoya

**Comment by alexander-held** [comment, marked as answer]
The most straightforward setup where this can arise is one where the up and down variations of a systematic behave the same way (e.g. both end up decreasing the yield consistently) and this behavior would consistently shift signal up or down. This is of course not the case for a `staterror`, where the up/down variations necessarily change the model prediction in different directions. It is certainly not impossible for this to happen, it could be for example that both the up and the down variation cause a pull in another parameter, and perhaps that other nuisance parameter consistently leads to more or less signal when pulled, no matter the direction (e.g. due to the `histosys` always having the same effect in a signal region for both up and down variations). For cases where the explanation is not as easy as up and down variations doing the same thing, I think it is a very model-specific question and I am not sure that it can be easily answered in general. To look more into it, you could manually fix that nuisance parameter at +/- 1 sigma and see what else moves around in the fit in response to that.

**Reply by Tomoya-Iizawa** [reply]
Thank you! that is really useful to have guiding principle for investigation.
Tomoya
